src/main/java/Echo.java

```diff
@@ -0,0 +1,175 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Echo {
+    static String break_line = "----------------------------------------";
+
+    public void startEchoing() {
```

Comment:
Please consider refactoring parts of this method to separate methods to adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)